### ISSUE TITLE
Use postcss warnings to unfixable bugs

### ISSUE THREAD

**Issue by kadmil** [luisrudge/postcss-flexbugs-fixes#17]
There are things we can't fix on css-only level. For such problems we could generate warnings with postcss API https://github.com/postcss/postcss/blob/master/docs/api.md#warning-class

Thank @ai for suggesting this. )

**Comment by luisrudge** [comment]
You mean for bugs that we can't evaluate if width or height, for example?

**Comment by kadmil** [comment]
Yep, like this. Or if we detect config which is known to cause bugs and can't fix them on css-only level
https://github.com/philipwalton/flexbugs/blob/master/README.md#2-column-flex-items-set-to-align-itemscenter-overflow-their-container

**Comment by MoOx** [comment]
:+1:

**Comment by cvrebert** [comment]
Can this qualify as a flexbug for the purposes of warning about it?: https://drafts.csswg.org/css-flexbox/#item-margins
> Authors should avoid using percentages in paddings or margins on flex items entirely, as they will get different behavior in different browsers.
